### DISCUSSION TITLE
Queries on - Microsoft AD integration, RDP server and management tools

### DISCUSSION THREAD

**Discussion by ebym** [Nitrux/nitrux-bug-tracker#165] [Q&A]
Q1: Is there a guide for desktop integration with Microsoft Active Directory ?.

Ubuntu have required tools and guides for AD integration, SSSD(System Security Services Daemon) and ADSys (extended Active Directory features, require Ubuntu Advantage Desktop subscription). Once SSSD is integrated, remote logins using AD credentials is possible xrdp server. 

Q2: Is there a wayland compatible RDP server that support headless sessions on Nitrux ?.

Q3: Is there cockpit/webmin like tool for managing few computers running Nitrux ?.

Thanks,

**Comment by UriHerrera** [comment]
Hi. 

1. I have never used Active Directory; if you have a guide you followed to set up Active Directory in Ubuntu, we can look at that.
2. What software were you using before? I know of KRDP, which is available as a Flatpak and claims to work with Wayland, but it doesn't have headless session support.
3. I'm also not familiar with these types of tools. However, I'm not sure that I understand the question. Are you asking if we develop similar software? Or how can you install it in the distribution? If the former, no, we don't make software like that. If the latter, you can install Cockpit in a container.

**Reply by ebym** [reply]
Hi,
1, I have integrated ubuntu systems with AD using SSSD(System Security Services Daemon) and ADSys(Ubuntu advantage subscription is required for ADSys to pushing Group Policies).  Ubuntu provides free [Integration guide here.](https://ubuntu.com/engage/microsoft-active-directory)

2, Have few Ubuntu MATE with xrdp, these have headless session support. 

3, cockpit/webmin are gui tools for linux server administration. Since Nitrux is to be deployed as Desktop, Do you have any Remote Monitoring and Management tools specifically for Nitrux on system level, rather than running in a container ?. 

Thanks,
